Common/KeyMap.cpp

```diff
@@ -318,9 +323,9 @@ void UpdateConfirmCancelKeys() {
 			confirmKeys.push_back(hardcodedConfirmKeys[i]);
 	}
 
-	const keycode_t hardcodedCancelKeys[] = {
-		NKCODE_ESCAPE,
-		NKCODE_BACK,
+	const KeyDef hardcodedCancelKeys[] = {
+		KeyDef(DEVICE_ID_KEYBOARD, NKCODE_ESCAPE),
+		KeyDef(DEVICE_ID_KEYBOARD, NKCODE_BACK),
```

Comment:
I added a DEVICE_ID_ANY to the PR to native and updated this PR to use this device ID.